section3/reflection.md

```diff
@@ -1,13 +1,28 @@
 ## Section 3 Reflection
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
-
+* I am not going to lie, it was a bit uncomfortable but also expected. I did spend time looking through past exercises, lessons, and re-watched some videos that were in our section plan. Lots of Google searches as well. I am glad I started section 3 after our last lesson and I found my notes to be valuable again to me.
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
-
+* I think Instagram uses Arrays to store information. Instagram might use Arrays to store information on how many followers a user might have on their profile, to how many posts a user might have, or how many likes a post has.
```

Comment:
YUP! Lots of Arrays in Instagram for sure. I'd bet they store the LIST of actual follows in the array (vs the "how many" in the array) - but maybe the use `array.length` to then find out how many followers a user has - does that make sense?